documentation/modules/upgrading/proc-upgrade-cli-tool-crds.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// assembly-upgrade-resources.adoc
+
+[id='proc-upgrade-cli-tool-crds-{context}']
+= Upgrading CRDs to {ApiVersion} using the API conversion tool
+
+[role="_abstract"]
+This procedure describes how to use the API conversion tool to convert the CRDs that define the schemas used to instantiate and manage Strimzi-specific resources in a format applicable to `{ApiVersion}`.
+To do so, you use the `crd-upgrade` command.
+
+Perform this procedure after converting your custom resources to `{ApiVersion}`.
```

Comment:
Can we stress this out by making it bold or something? Also, it might be worth making it clear tht it should be after converting all Strimzi custom resources in the whole Kube cluster.

Reply:
Bolded and expanded as suggested: _*Perform this procedure after converting all Strimzi custom resources in the whole Kubernetes cluster to `{ApiVersion}`.*_